src/main/java/org/ays/auth/port/AysRoleReadPort.java

```diff
@@ -18,6 +19,13 @@
  */
 public interface AysRoleReadPort {
 
+    /**
+     * Retrieves all roles.
+     *
+     * @return A set containing all roles.
+     */
+    Set<AysRole> findAll();
```

Comment:
Set dönmek yerine `List` dönebiliriz. MapStruct kütüphanesinde fark ettiğimiz bir problem vardı. Bu sebeple spesifik ihtiyacımız olmadığı sürece `Set` yerine `List` kullanmamız daha sağlıklı olabilir.

```suggestion
     * @return A list containing all roles.
     */
    List<AysRole> findAll();
```

Reply:
Değiştirdim

Reply:
değişikliği yaptım